2/contrib/jenkins/install-plugins.sh

```diff
@@ -347,7 +347,7 @@ function installedPlugins() {
 function jenkinsMajorMinorVersion() {
     if [[ -f "$JENKINS_WAR" ]]; then
         local version major minor
-        version="$($(alternatives --display java | awk '/family.*x86_64/ { print $1; }') -jar $JENKINS_WAR --version)"
+        version="$(/etc/alternatives/java -jar $JENKINS_WAR --version)"
```

Comment:
java is not in the path in the assemble step, it is set in the run script https://github.com/openshift/jenkins/blob/626247110ad9c23e7e238185fd0dcba539179499/1/contrib/s2i/run#L18-L28

32/64 bit does not matter for just getting the version of the war

Reply:
yep actually getting at java was also part of my reference in https://github.com/openshift/jenkins/pull/466#discussion_r159677972 @bakito - apologies if it was not clear

and yeah, if you've got an alternatives usage with your latest commit that does not awk on the family, all the better, and you do not have to be concerned with the 32 vs. 64 bit questions

I'll initiate a test momentarily.  Assuming it passes, we'll ask that you squash the commits and we'll merge.